developers/weaviate/installation/docker-compose.md

```diff
@@ -5,30 +5,32 @@ image: og/docs/installation.jpg
 # tags: ['installation', 'Docker']
 ---
 
+Weaviate supports deployment with Docker. Starting in v1.24.0, there is an image that runs using default values. Alternatively, edit the `docker-compose.yml` file to customize your instance. 
 
-## Overview
+## Default Weaviate environment
 
-Weaviate supports deployment with Docker Compose, which allows you to run Weaviate on any OS supported by Docker.
+:::info Added in v1.24.0
 
-To start Weaviate with Docker, you can use a Docker Compose file, typically called `docker-compose.yml`. You can:
-* use the [Starter Docker Compose file](#starter-docker-compose-file),
-* generate one with the [configuration tool](#configurator),
-* pick one of the [examples](#example-configurations) below.
-
-:::note
-If you are new to Docker (Compose) and containerization, check out our [Docker Introduction for Weaviate Users](/blog/docker-and-containers-with-weaviate).
 :::
 
-## Starter Docker Compose file
+The default docker image doesn't need any configuration. To run a basic Weaviate instance, run this command from a terminal:
 
-:::info Starter Docker Compose
-If you are new to Weaviate, this is a good place to start.
-:::
+```bash
+ docker run -p 8080:8080 semitechnologies/weaviate:1.24.0
```

Comment:
Hey @daveatweaviate I think this needs to be 
```
docker run -p 8080:8080 -p 50051:50051 semitechnologies/weaviate:1.24.0
```

Otherwise the gRPC traffic will not be passed on to Weaviate.

Also if we use `semitechnologies/weaviate:||site.weaviate_version||` I believe it will update the version at build time.